python/feathub/common/utils.py

```diff
@@ -88,6 +88,24 @@ def append_unix_time_column(
     return unix_time_column
 
 
+def append_epoch_millis_column(
+    df: pd.DataFrame, timestamp_field: str, timestamp_format: str, tz: tzinfo
+) -> str:
+    epoch_millis_column = "_epoch_millis_time"
```

Comment:
Would it be more consistent with other processors to use `EVENT_TIME_ATTRIBUTE_NAME`?